### PR TITLE
SEO-184285-UWP-Overview-Parmanent-Redirect-link

### PR DESCRIPTION
**Pull request by Faith-Atieno**
Title | Description
-- | --
|Task Category | Site Audit work|
|Content Task Link/Mail Screenshot | NA|
|Hotfix |https://github.com/syncfusion-content/uwp-docs/pull/660|
|Ticket/Task link |https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/184285|
|Work link |https://syncfusion.sharepoint.com/:x:/r/sites/SEOWebsiteAuditing/_layouts/15/Doc.aspx?sourcedoc=%7B0C07BBB1-1233-47A7-818C-275A98FD0063%7D&file=Faith-%20Site%20Audit.xlsx&action=default&mobileredirect=true|

Title | Description
-- | --
|Task Category | Coverage Issues|
|Content Task Link/Mail Scr
Changes made | Reason for change
-- | --
| Page with redirect| To increase page performance |

### CODE REVIEW

**Comment by Faith-Atieno** [comment]
rebuild

File: uwp/Overview.md [hunk 1]
```diff
@@ -348,7 +348,7 @@ If you are still unable to find the information that you are looking for in the
 			FILE FORMATS<br/><br/>
 		</td>
 		<td>
-			<a href="/file-formats/docio/overview">DocIO</a><br/><br/>
+			<a href="/word-library/net/overview">DocIO</a><br/><br/>
```
Comment (Yvone-Atieno) [inline uwp/Overview.md:351]: Revert the changes made in this md

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: uwp/Overview.md (hunk 1 above)
Comment (Faith-Atieno) [reply uwp/Overview.md]: Changes reverted hence closing this PR

**Pull request closed without merging**